src/components/S/IDuration.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+const props = defineProps<{
+  modelValue?: number;
+  error?: string;
+  definition: any;
+}>();
+
+const emit = defineEmits<{
+  (e: 'update:modelValue', value: number);
+}>();
+
+const DAYS = new Array(31 + 1).fill(0).map((_, i) => i);
+const HOURS = new Array(24).fill(0).map((_, i) => i);
+const MINUTES = new Array(60).fill(0).map((_, i) => i);
+
+const days = ref(0);
+const hours = ref(0);
+const minutes = ref(0);
+
+watch([days, hours, minutes], () => {
+  const value = (days.value * 24 * 60 + hours.value * 60 + minutes.value) * 60;
+  emit('update:modelValue', value);
+});
+
+watch(
```

Comment:
Why not use computed with get and set instead of 2 watchers? Example here: https://github.com/snapshot-labs/tune/blob/7e2a1558768f5142701dafb60d181025039c6dfc/src/components/TuneInputDuration.vue#L19

Reply:
I think that using computed here could possibly simplify our watchers (don't think they are that bad anyway), but it will complicate how we handle our individual inputs.